syntax = "proto3";

package sample;

option java_package = "sample.proto"; 
option java_multiple_files = true; 
option java_outer_classname = "UserProfileProto";


message UserProfile {
  string user_id = 1;
  string display_name = 2;
  int32 age = 3;
  bool is_active = 4;
  repeated string interests = 5;
}